pyproject.toml

```
@@ -6,7 +6,7 @@ license = "MIT"
 name = "stackstac"
 readme = "README.md"
 repository = "https://github.com/gjoseph92/stackstac"
-version = "0.2.1"
+version = "0.2.2"
```

```suggestion
version = "0.2.1"
```
I'd prefer to bump the stackstac version as a separate commit if that's okay?